src/OEBPS/Text/titlepage.xhtml

```diff
@@ -12,13 +12,13 @@
 
   <hr class="narrow"/>
 
-  <p class="title">Sigil User Guide</p>
+  <h1 class="title">Sigil User Guide</h1>
 
-  <p class="version">For Sigil Versions 2.3.0 and Later</p>
+  <h2 class="version">For Sigil Versions 2.3.0 and Later</h2>
 
```

Comment:
I'm ok with the style changes (especially after the new commit), but won't the title and version lines being changed from p to h1 cause new entries to be added to the generated TOC, ncx, nav, etc... ? Or am I missing something?


Unless I'm missing something